clientv3/client.go

```diff
@@ -200,5 +200,6 @@ func dialEndpointList(c *Client) (*grpc.ClientConn, error) {
 }
 
 func isRPCError(err error) bool {
-	return grpc.Code(err) != codes.Unknown
+	return grpc.Code(err) != codes.Unknown ||
+		err == grpc.ErrClientConnClosing
```

Comment:
OK, it looks like it's timing out on Get and the change breaks out of it but strangely Get isn't returning an error like it should.